license-compliance/gradle-license-report/info-licenses.json

```diff
@@ -0,0 +1,404 @@
+      "licenseCategory": ""
+    },
+    {
+      "bundleName": "lpgl20-mit",
+      "licenseName": "LGPL-2.0 OR MIT",
+      "licenseUrl": "https://spdx.org/licenses/LGPL-2.0-only.html OR https://spdx.org/licenses/MIT.html",
+      "licenseCategory": ""
+    },
+    {
+      "bundleName": "cc0-1-mit",
+      "licenseName": "MIT OR CC0-1.0",
+      "licenseUrl": "https://spdx.org/licenses/MIT.html OR https://spdx.org/licenses/CC0-1.0.html",
+      "licenseCategory": ""
+    },
+    {
+      "bundleName": "mit-0",
+      "licenseName": "MIT-0",
+      "licenseUrl": "https://spdx.org/licenses/MIT-0.html",
+      "licenseCategory": "Permissive"
+    },
+    {
+      "bundleName": "lgpl2.1-or-later",
+      "licenseName": "LGPL-2.1+",
+      "licenseUrl": "https://spdx.org/licenses/LGPL-2.1+.html",
+      "licenseCategory": ""
+    },
+    {
+      "bundleName": "apache2",
+      "licenseName": "Apache-2.0",
+      "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0.txt",
+      "licenseCategory": "Permissive"
+    },
+    {
+      "bundleName": "bsd2clause",
+      "licenseName": "BSD-2-Clause",
+      "licenseUrl": "https://opensource.org/licenses/BSD-2-Clause",
+      "licenseCategory": "Permissive"
+    },
+    {
+      "bundleName": "bsd3clause",
+      "licenseName": "BSD-3-Clause",
+      "licenseUrl": "https://opensource.org/licenses/BSD-3-Clause",
+      "licenseCategory": "Permissive"
+    },
+    {
+      "bundleName": "cc0-1",
+      "licenseName": "CC0-1.0",
+      "licenseUrl": "http://creativecommons.org/publicdomain/zero/1.0/",
+      "licenseCategory": "Public Domain"
+    },
+    {
+      "bundleName": "cddl1",
+      "licenseName": "CDDL-1.0",
+      "licenseUrl": "https://opensource.org/licenses/CDDL-1.0",
+      "licenseCategory": "Weak Copyleft"
+    },
+    {
+      "bundleName": "epl2",
+      "licenseName": "EPL-2.0",
+      "licenseUrl": "https://www.eclipse.org/legal/epl-2.0/",
+      "licenseCategory": "Weak Copyleft"
+    },
+    {
+      "bundleName": "gpl2.0",
+      "licenseName": "GPL-2.0",
+      "licenseUrl": "https://spdx.org/licenses/GPL-2.0.html",
+      "licenseCategory": ""
+    },
+    {
+      "bundleName": "lgpl2.1",
+      "licenseName": "LGPL-2.1",
+      "licenseUrl": "https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html",
+      "licenseCategory": "Weak Copyleft"
+    },
+    {
+      "bundleName": "mit",
+      "licenseName": "MIT",
+      "licenseUrl": "http://www.opensource.org/licenses/mit-license.php",
+      "licenseCategory": "Permissive"
+    },
+    {
+      "bundleName": "mpl1.1",
+      "licenseName": "MPL-1.1",
+      "licenseUrl": "https://spdx.org/licenses/MPL-1.1.html",
+      "licenseCategory": "Weak Copyleft"
+    },
+    {
+      "bundleName": "upl1.0",
+      "licenseName": "UPL-1.0",
+      "licenseUrl": "https://spdx.org/licenses/UPL-1.0.html",
+      "licenseCategory": "Permissive"
+    }
+  ],
+  "transformationRules": [
```

Comment:
question: Do we need this block here? I believe transformation rules are not required for getting licence and `licenseCategory` in your check_license script

Reply:
The transformationRules block here looks redundant because the script use transformationRules from the license-normalizer-bundle.json, also the same file is used in gradle plugin
If we decide to remove the block from this file, could you please compare its content to the transformationRules block in the license-normalizer-bundle.json file because they are different